presto-docs/src/main/sphinx/sql/set-session.rst

```diff
@@ -13,14 +13,36 @@ Synopsis
 Description
 -----------
 
-Set a session property value.
+Set a session property value or a catalog session property.
+
+Catalog session properties use the name of a catalog as a prefix in front of a
+session property supported by a connector.
 
 Examples
 --------
 
+The following example enables optimized hash generation.
+
 .. code-block:: sql
 
     SET SESSION optimize_hash_generation = true;
+
+An example of the usage of catalog session properties is the :doc:`Accumulo
+connector </connector/accumulo>`, which supports a property named
+``optimize_locality_enabled``. If you use the connector in a catalog properties
+file named ``keyvaluestore.properties``, you need to use the following command
+to set the catalog session property.
+
+.. code-block:: sql
+
+    SET SESSION keyvaluestore.optimize_locality_enabled = false;
+
+This catalog session property does not apply to any other catalog, even if it
+also uses the Accumulo connector.
+
+Another example is the setting the ``optimized_reader_enabled`` for the catalog
```

Comment:
Let's remove this example, since the property no longer exists, and the example doesn't seem to provide any additional value. The above is much better than this old example.

Reply:
done